tools/crm_mon.c

```diff
@@ -515,6 +533,20 @@ main(int argc, char **argv)
         one_shot = TRUE;
     }
 
+    /* to enable stonith-connection when called via some application like pcs 
+     */
+    if (getenv("FENCE_HISTORY") != NULL) {
+        fence_history = TRUE;
+        fence_connect = TRUE;
+        show |= mon_show_fence_history;
+    }
+    if (getenv("FENCE_FULL_HISTORY") != NULL) {
+        fence_full_history = TRUE;
+        fence_history = TRUE;
+        fence_connect = TRUE;
+        show |= mon_show_fence_history;
+    }
```

Comment:
In the end not at all ...
While it doesn't make use of the new section in the xml-output at least it is an easy way to control the creation of this section through pcs. See the cmd-example above. I used it e.g. to check if the schema-adaption makes pcs accept the xml without having to touch the pcs-code.

Reply:
Makes sense. I think I'd rather pull this out before release. I prefer to avoid environment variables except when there's no better way (e.g. settings that pacemaker needs at daemon start-up).

Reply:
With the new --fence-history[=level] it degraded to a single line. I would comment it out on the release.